src/win/tty.c

```diff
@@ -2123,9 +2123,9 @@ static int uv_tty_write_bufs(uv_tty_t* handle,
       }
 
       /* We wouldn't mind emitting utf-16 surrogate pairs. Too bad, the windows
-       * console doesn't really support UTF-16, so just emit the replacement
-       * character. */
-      if (utf8_codepoint > 0xffff) {
+       * console before Windows 10 doesn't really support UTF-16, so just emit
+       * the replacement character. */
+      if (!is_windows_10_or_greater && utf8_codepoint > 0xffff) {
         utf8_codepoint = UNICODE_REPLACEMENT_CHARACTER;
       }
```

Comment:
```suggestion
```

The proposal by Dustin and myself is to also eliminate this restriction entirely.

Reply:
The question is, does it make sense to pass them through below windows 10? Dustin suggested that it works all the way back to 
>  10.0.14393 (the earliest version I have in a VM)

Reply:
Yes, that's just the version number for the OS he was running, but it's always been possible to run an arbitrary application on the OS which support this.

Reply:
bump?